app/views/admin/take_part_pages/index.html.erb

```diff
@@ -1,38 +1,59 @@
-<% page_title 'Get involved' %>
+<% content_for :page_title, "Get involved" %>
+<% content_for :title, "Get involved" %>
+<% content_for :title_margin_bottom, 4 %>
 
-<div class="get-involved-header">
-  <h1>Get involved</h1>
-  <%= link_to "View on website", get_involved_url %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p class="govuk-body">
+      <%= link_to "View on website", get_involved_url, class: "govuk-link", target: "_blank" %>
+    </p>
+  </div>
 </div>
 
-<section class="get-involved">
-  <%= get_involved_tab_navigation %>
-
-  <h2>Take part pages</h2>
-  <p class="warning">
-    Warning: changes to take part page ordering appears instantly on the live site.
-  </p>
+<%= render "components/secondary_navigation", {
```

Comment:
On other pages that use 2/3 width this is in the 2/3 column. I think we need to do that here

Reply:
Yes, I think that's correct. Actually it isn't that way in the design for the page but it is more consistent, not least with the "Get involved" tab.